lib/packwerk/cache.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+# typed: strict
+
+module Packwerk
+  class Cache
+    CACHE_DIR = T.let(Pathname.new("tmp/cache/packwerk"), Pathname)
```

Comment:
I didn't realize people would be `cd`ing into a specific package... I wouldn't expect `bundle exec` to succeed unless that package has a `Gemfile` (e.g. it's a gem that also has packwerk installed/maintained). I figured this would only be run through the root. Either way though -- I think having a configurable cache directory makes a ton of sense. Different people may set up their machines differently, and I could see different repos wanting a different cache location, so this is changed!